gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -1198,6 +1198,8 @@ gst_tensor_filter_detect_framework (const gchar * const *model_files,
       detected_fw = g_strdup ("python");
     else if (g_str_equal (ext[0], ".graph"))
       detected_fw = g_strdup ("movidius-ncsdk2");
+    else if (g_str_equal (ext[0], ".ini"))
```

Comment:
IMO, since the extension names such as ```ini``` are too broadly used, it would be better to add more conditions to check whether the given file is for entrainer or not.

Reply:
I agree that it is quite broad. Could you give me a suggestion to detect the framework? 
or could you recommend other supplementary choice?
1. to not support auto detection for now and comment this section
2. to warn user that it is selected as nntrainer but it might not be intended

Reply:
If other frameworks use the same .ini extension later, it would be good to use priority like tflite.

Reply:
*.nntr? or *.nnt?

Reply:
`.nnt` is already in use.
PTAL, https://fileinfo.com/extension/nnt